web/server/api/endpoints/lineage.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import annotations
+
+import collections
+import traceback
+import typing as t
+
+from fastapi import APIRouter, Depends, HTTPException
+from sqlglot import exp, parse_one
+from sqlglot.lineage import Node, lineage
+from starlette.status import HTTP_422_UNPROCESSABLE_ENTITY
+
+from sqlmesh.core.context import Context
+from web.server.settings import get_loaded_context
+
+router = APIRouter()
+
+
+def _get_table(node: Node) -> str:
+    """Get a node's table/source"""
+    if isinstance(node.expression, exp.Table):
+        return (
+            f"{node.expression.db}.{node.expression.this}"
+            if node.expression.db
+            else str(node.expression.this)
+        )
+
+    else:
+        return node.alias
+
+
+def _process_downstream(downstream: t.List[Node]) -> t.Dict[str, t.List[str]]:
+    """Aggregate a list of downstream nodes by table/source"""
+    graph = collections.defaultdict(list)
+    for node in downstream:
+        column: exp.Column = parse_one(node.name)
```

Comment:
column = exp.column(node.name) why do you need a column here?

Reply:
I don't need a column here per se. I need the column name without the table for this node. For example, node.name could be the string, "oi.quantity", and I want to extract "quantity" from it.